exploration/text-vs-code.md

```diff
@@ -28,38 +32,69 @@ ICU MessageFormat and Fluent both support inline selectors
 separated from the text using `{…}` for multi-variant messages.
 ICU MessageFormat is the only known format that uses `{…}` to also delimit text.
 
-[Mustache templates](https://mustache.github.io/mustache.5.html)
-and related languages wrap "code" in `{{…}}`.
-In addition to placeholders that are replaced by their interpolated value during formatting,
-this also includes conditional blocks using `{{#…}}`/`{{/…}}` wrappers.
-
-[Handlebars](https://handlebarsjs.com/guide/) extends Mustache expressions
-with operators such as `{{#if …}}` and `{{#each …}}`,
-as well as custom formatting functions that become available as e.g. `{{bold …}}`.
-
-[Jinja templates](https://jinja.palletsprojects.com/en/3.1.x/templates/) separate
-`{% statements %}` and `{{ expressions }}` from the base text.
-The former may define tests that determine the inclusion of subsequent text blocks in the output.
-
-A cost that the message formatting and templating languages mentioned above need to rely on
-is some rule or behaviour that governs how to deal with whitespace at the beginning and end of a pattern,
-as statements may be separated from each other by newlines or other constructs for legibility.
+Formatting and templating are distinct operations with similarities.
+Both interpolate strings by using input values,
+provided as inputs alongisde the formatting pattern string or template,
+to produce a new string.
```

Comment:
I've been trying to back away from the use of "interpolate" and it strikes me that this could be said more simply. What do you think of:

```suggestion
Both take input values to replace portions of the pattern string or template, producing a new, formatted, string.
```